okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/ConcurrencyLimiters.java

```diff
@@ -209,6 +215,10 @@ public void onDropped() {}
 
         synchronized void processQueue() {
             while (!waitingRequests.isEmpty()) {
+                log.debug("Limit",
+                        SafeArg.of("limit", limiter.getLimit()),
+                        safeArgMethod,
+                        safeArgPathTemplate);
```

Comment:
should we put this under a `log.isDebugEnabled()` guard so we don't create an object each time?

Reply:
I dunno man there are so many objects that get allocated when you make a HTTP request already, i've already pulled out the method and the path template because they're constant, I think `SafeArg.of("limit", limiter.getLimit())` isn't thaaat bad